docs/install/source.md

```diff
@@ -0,0 +1,314 @@
+        ```
+    
+    === "RHEL / CentOS"
+
+        1. Install the dependencies. The following command installs the dependencies for CentOS 8:
+
+            ```{.bash data-prompt="$"}
+            $ sudo yum -y install centos-release-scl epel-release 
+            $ sudo yum -y install python38 python38-devel python38-pip python3-scons gcc-toolset-9 gcc-c++ gcc-toolset-11-dwz gcc-toolset-11-elfutils cmake3 wget openssl-devel zlib-devel cyrus-sasl-devel xz-devel bzip2-devel libcurl-devel lz4-devel e2fsprogs-devel krb5-devel openldap-devel expat-devel cmake make
+            ```
+
+        2. Build a specific ``curl`` version
+
+            -  Fetch the package archive  
+                ```{.bash data-prompt="$"}
+                $ wget https://curl.se/download/curl-7.66.0.tar.gz
+                ```  
+            -  Unzip the package  
+                ```{.bash data-prompt="$"}
+                $ tar -xvzf curl-7.66.0.tar.gz && cd curl-7.66.0
+                ```  
+            -  Configure and build the package  
+                ```{.bash data-prompt="$"}
+                $ ./configure
+                $ sudo make install
+                ```
+
+5. Build the AWS Software Development Kit for C++ library
+
+    -  Create a directory to store the AWS library 
+
+        ```{.bash data-prompt="$"}
+        $ mkdir -p /tmp/lib/aws
+        ``` 
+
+    -  Declare an environment variable ``AWS_LIBS`` for this directory 
+
+        ```{.bash data-prompt="$"}
+        $ export AWS_LIBS=/tmp/lib/aws
+        ``` 
+
+    -  Percona Server for MongoDB is built with AWS SDK CPP 1.8.187
+       version. Switch to this version 
+
+        ```{.bash data-prompt="$"}
+        $ cd aws-sdk-cpp && git checkout 1.8.187
+        ``` 
+
+    -  It is recommended to keep build files outside the SDK directory.
+       Create a build directory and navigate to it 
+
+        ```{.bash data-prompt="$"}
+        $ mkdir build && cd build
+        ``` 
+
+    -  Generate build files using ``cmake`` 
+
+        ```{.bash data-prompt="$"}
+        $ cmake .. -DCMAKE_BUILD_TYPE=Release -DBUILD_ONLY="s3;transfer" -DBUILD_SHARED_LIBS=OFF -DMINIMIZE_SIZE=ON -DCMAKE_INSTALL_PREFIX="${AWS_LIBS}"
+        ``` 
+
+    -  Install the SDK 
+
+        ```{.bash data-prompt="$"}
+        $ make install
+        ```
+
+6. Build Percona Server for MongoDB
+
+    -  Change directory to ``percona-server-mongodb`` 
+
+        ```{.bash data-prompt="$"}
+        $ cd percona-server-mongodb
+        ``` 
+
+    -  Build Percona Server for MongoDB from ``buildscripts/scons.py``.
+        
+        ```{.bash data-prompt="$"}
+        $ buildscripts/scons.py -j$(nproc --all) --jlink=2 --disable-warnings-as-errors --ssl --opt=on --use-sasl-client --wiredtiger --audit --inmemory --hotbackup CPPPATH="${AWS_LIBS}/include" LIBPATH="${AWS_LIBS}/lib" install-mongod
+        ``` 
+
+       This command builds only the database. Other available targets for the
+       ``scons`` command are:  
+
```

Comment:
all those targets must include "install-" prefix in v6.0.

Reply:
install-mongod
install-mongos
and so on..

Reply:
Updated
Added the build instructions for FIPS